packages/components/src/components/alert/alert.tsx

```diff
@@ -167,6 +167,13 @@ export class Alert extends LitElement {
   /** Specifies the size of the component. */
   @property({ reflect: true }) scale: Scale = "m";
 
+  /**
+   * When true, disables top layer placement when the component is open.
+   *
+   * @mdn [Top Layer](https://developer.mozilla.org/en-US/docs/Glossary/Top_layer)
```

Comment:
I wonder if we should add a note to not disable the top layer unless x,y,z? Most users shouldn't touch this. Maybe we can be specific about when enabling this property would be useful or necessary?